pygmt/src/grdimage.py

```diff
@@ -161,6 +163,19 @@ def grdimage(self, grid, **kwargs):
     {perspective}
     {transparency}
     {cores}
+
+    Example
+    -------
+    >>> import pygmt
+    >>> # load the 30m grid with "gridline" registration
+    >>> grid = pygmt.datasets.load_earth_relief("30m", registration="gridline")
```

Comment:
Maybe we should state the unit in long form in the comment?
```suggestion
    >>> # load the 30 arc minute grid with "gridline" registration
    >>> grid = pygmt.datasets.load_earth_relief("30m", registration="gridline")
```